Shittu1/fast-food-fast
# Database setup #160919437

** What does this PR do?**
Connects users APIs to database

**Description of Task to be completed?**
- create users APIs
- setup database
- put in place authentication and authorization for signups

**How should this be manually tested?**
- test APIs with postman.
- run the pg admin app and ensure that the database is set up.

**What are the relevant pivotal tracker stories?**
- Set up postgreSQL database - #160919437
